Makefile

```diff
@@ -1,14 +1,64 @@
-build: clean
-	GOARCH=amd64 GOOS=linux go build -ldflags "-s -w" -o ./bin/dicterm cmd/dicterm/main.go
+prefix?=$(GOPATH)/bin
+root?=$(shell pwd)
+name?=$(notdir $(root))
+srcpath?=$(root)/cmd/$(name)
+builddir?=$(root)/bin
 
-clean:
+
+
+goos?=$(shell go env GOOS)
+arch?=$(shell go env GOARCH)
+ldflags?=-ldflags "-s -w"
+
+
+
+mansrc?=$(root)/README.md
+ifeq ($(shell uname),Linux)
+	mandir?=/usr/local/man/man1
```

Comment:
`manpath` does not include this path, why did you choose this path?

```shell
➜  dicterm git:(makefile) ✗ make   
go mod tidy
go clean
mkdir -p /home/murat/dev/dicterm/bin
GOOS=linux GOARCH=amd64 go build -ldflags "-s -w" -o /home/murat/dev/dicterm/bin/dicterm /home/murat/dev/dicterm/cmd/dicterm
make: *** No rule to make target '/usr/local/man/man1', needed by 'man'.  Stop.
➜  dicterm git:(makefile) ✗ manpath
/home/murat/.nvm/versions/node/v14.18.2/share/man:/usr/local/man:/usr/local/share/man:/usr/share/man
➜  dicterm git:(makefile) ✗ uname
Linux
```

Reply:
I misread a howtogeek article months ago, and because I've never actually tested this on linux (I'm on osx) I'm only just finding that out.

Does it not show up at all when you `man dicterm`? It might, because `/man1` is just a section of your manpages (though I'm assuming it's even laid out like that on your system).

Do you have write access to `/usr/local/man` by default, or would you need sudo/doas or something?

Also, when you put something in there, does calling`man something` work immediately or do you _need_ to update the database first?


article: https://www.howtogeek.com/682871/how-to-create-a-man-page-on-linux/

Reply:
You can list manpage paths via `manpath` command as far as I know. But actually, I'm not sure about putting the readme file as a manpage, it does not look like a manpage.